ffmuc-mesh-vpn-wireguard-vxlan/check_site.lua

```diff
@@ -0,0 +1,11 @@
+local function check_peer(k)
+	-- we do need unnamed / numeric keys
+	need_string_match(in_domain(extend(k,
+		{'publickey'})), "^" .. ("[%a%d+/]"):rep(42) .. "[AEIMQUYcgkosw480]=$")
```

Comment:
This is probably taken from here:
https://stackoverflow.com/a/74438437

sounds good.

Reply:
Actually, no. This is from https://github.com/freifunk-gluon/gluon/blob/d4847b697c1fb9da44751d659072e77e4dae6967/package/gluon-mesh-vpn-wireguard/check_site.lua#L5